### ISSUE TITLE
Helm Chart

### ISSUE THREAD

**Issue by plumdog** [kubernetes-sigs/aws-iam-authenticator#198]
I've created a [PR for a helm chart](https://github.com/helm/charts/pull/10940) to install the daemonset. This is pretty directly taken from https://github.com/kubernetes-sigs/aws-iam-authenticator/blob/master/example.yaml and just turned into a chart.

This issue is to:
- check that me doing this is fine
- give anyone who knows more about the manifests a chance to review that PR or make future alterations to the chart
- act as a pointer for anyone who finds this repo and is looking for a helm chart for it

**Comment by christopherhein** [comment]
Seems like it's already been merged, as things progress will you be maintaining it?

**Comment by plumdog** [comment]
Hi @christopherhein,

Indeed, the chart is now merged - see https://github.com/helm/charts/tree/master/stable/aws-iam-authenticator - and can be installed with `helm install stable/aws-iam-authenticator`.

I'm listed as a maintainer in the `Chart.yaml` metadata, so I'll get notified when anyone opens a PR to change the chart. But I don't have any plans to monitor aws-iam-authenticator itself for changes that I think would require a change to the chart. I guess if someone creates an issue about it in https://github.com/helm/charts or here and mentions me, I'd be happy to take a look.

In short: I'll try my best at doing reactive maintenance on the chart, but I have no plans to do proactive maintainance.

**Comment by christopherhein** [comment]
Okay, good to know. Main reason I ask is as of last week we released the 0.4.0 release, which has a new Docker registry and a couple changed flags.

**Comment by austince** [comment]
I would love to see this maintained - if I had the bandwidth I would chip in, but just wanted to say I appreciate the initial effort!

**Comment by christopherhein** [comment]
This was my concern as well. Potentially moving the chart into the project would help.

**Comment by plumdog** [comment]
Per https://github.com/helm/charts/issues/17126, @nckturner is now a maintainer of the chart in stable. I suspect this will essentially supersede this issue.
